tests/unit/backend/layers/business/test_business.py

```diff
@@ -151,6 +151,11 @@ def mock_config_fn(name):
             donor_id=["test_donor_1"],
             is_primary_data="BOTH",
             x_approximate_distribution="normal",
+            default_embedding="X_embedding_1",
```

Comment:
If we were using pytest fixture this would be easy to factor out and avoid repeating.